sklearn/feature_extraction/text.py

```diff
@@ -1307,6 +1312,12 @@ class TfidfVectorizer(CountVectorizer):
 
     Equivalent to CountVectorizer followed by TfidfTransformer.
```

Comment:
please put backticks around the class names, so they will be hyperlinked in the online docs.